snakemake_executor_plugin_aws_batch/__init__.py

```
@@ -294,19 +148,14 @@ def run_job(self, job: JobExecutorInterface):
         self.report_job_submission(
             SubmittedJobInfo(
                 job=job,
-                external_jobid=submitted["jobId"],
+                external_jobid=job_info["jobId"],
                 aux={
                     "jobs_params": job_params,
```

Ruff (0.8.2)

Undefined name `job_params`

(F821)